### DISCUSSION TITLE
Git-Modes 1.4.0 released

### DISCUSSION THREAD

**Discussion by tarsius** [magit/git-modes#154] [Show and tell]
Previously the contents of the repository were available as three separate packages `gitattributes-mode`, `gitconfig-mode` and `gitignore-mode`.  These packages have been removed from Melpa and you should
remove them locally too and then install `git-modes` (either from Melpa or NonGNU Elpa), but make sure that it is already available before you do.  That can take a few hours.

See the [announcement](https://emacsair.me/2021/10/21/git-modes) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/magit/git-modes/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by luisalduucin** [comment]
Hi !! How do I remove the now missing packages and update to this new one? Or is the `git` layer going to update alone sometime?

**Reply by tarsius** [reply]
I assume you are talking about the git layer of some starterkit. I am sure it will be updated eventually. Maybe that has already happened. There's bound to be an issue about it in the issue tracker of the starterkit you happen to use. You should go look there.

**Reply by tarsius** [reply]
So probably https://github.com/syl20bnr/spacemacs/issues/15111 or https://github.com/hlissner/doom-emacs/issues/5667, as well as issues referenced there.

**Reply by luisalduucin** [reply]
Got it ! Thanks.
